component-library/components/sections/Left-right-block--Featured-map/Left-right-block--Featured-map.scss

```
@@ -0,0 +1,77 @@
+.c-contact-block {
+    $c: &;
+    display: grid;
+    grid-template-columns: var(--twelve-column-grid);
+    row-gap:48px;
+    padding:96px 0 0!important;
+  
+    &__content {
+      padding: 0;
+      grid-column: 2 / -2;
+    }
+  
+    &__content-text {
+      margin-bottom: 1.5rem;
+      max-width: 32.0625rem;
+      text-wrap: pretty;
+    }
+  
+    &__content-wrapper {
+      border-left: 0.0625rem solid var(--brand-grey-000);
+      padding-left: 1.5rem;
+      display:flex;
+      flex-direction:column;
+      align-items: flex-start;      
+
+      .c-social-icons, .c-labelled-icon{
+        justify-content:flex-start;
+      }
+    }
+
+    &__contact-details, &__content{
+      display:flex;
+      flex-direction: column;      
+    }
+        
+    &__contact-details{ gap: 24px; }
+    &__content{ gap:8px; }
+   
+    &__map-wrapper {
+      aspect-ratio: 1 / 1;
+      object-fit: cover;
+      grid-column: 1 / -1;
+      width:100%;     
+      max-height:700px; 
+
+      & > div{ height:100%; }
```

```suggestion
      & > .c-google-map-embed { height:100%; }
```
This does require the class being added to all parts of the if loop in the subcomponent (which they probably should have anyway)

---

Looks like this is not needed anyway - I believe my map was not 100% because of the missing API key - the code appears to have height:100% on it for when the key is present, so I assume this was a decision to have the map not stretch 100% when no key is present?

- removed the style altogether - maybe revisit the 'map embed' component later? seems a separate issue to this component